### DISCUSSION TITLE
how to replace system clock

### DISCUSSION THREAD

**Discussion by MartijnJonkers** [martinet101/ElevenClock#333] [Q&A] [answered]
I really like the feature where I get the seconds. But I would like to also have the seconds om my main display, should the latest update make this possible? I cannot find any settings for this.

Great tool, though!

**Comment by marticliment** [comment, marked as answer]
You need to enable select a custom color under appearance, and click on the bottom-right corner of the screen. Let me show:
![image](https://user-images.githubusercontent.com/53119851/143763692-25598c67-c62e-4c02-865c-53489b2a120d.png)
For step 4, click anywhere on the taskbar background (next to the white square)
![image](https://user-images.githubusercontent.com/53119851/143763715-e1dfb624-4718-4e28-8c6e-4c097e1ac531.png)

**Reply by MartijnJonkers** [reply]
that works for me! thanks!

**Reply by TurtleP** [reply]
This is a great solution for static wallpapers, but I myself have a slideshow that I use for my monitors. Forcing a specific background color in this situation does not work since it'll still use the Mica blended color from the initial wallpaper. What should I do in this instance?

**Reply by marticliment** [reply]
One solution I can recommend is setting your accent color as background color, which actually looks fine (unless your accent color is dynamic, which then is an issue)

**Reply by TurtleP** [reply]
This isn't really a solution that I would personally like, as I'd rather be able to keep using the Mica colors on the Taskbar. If there was a way for ElevenClock to grab the background color around that area automatically, I think that would be beneficial.

**Reply by marticliment** [reply]
I'm working on adding a background like the context menu's one on the clock, but for the moment it looked really weird, i'm still improving it

**Reply by TurtleP** [reply]
Yeah, no worries! I figured that this was something I should point out in case anyone else came across a similar situation.

**Reply by MartijnJonkers** [reply]
I now see an issue while using this. When a new notification is pending, the underlying clock keep peeking out: When I click the clock to view the notification, it moved back behind elevenclock.
![image](https://user-images.githubusercontent.com/606644/145980810-c7e8f1c7-ecb2-43f6-904c-6431a8466a54.png)

Is it possible to let elevenclock also show the new notification indicator? or something like that?

**Reply by marticliment** [reply]
This is an open issue, check it out

**Reply by Dypsis** [reply]
When I enable seconds, ElevenClock keeps expanding and contracting in width because of the changing seconds and partially obscures the Speaker icon.

Edit: I am experimenting with a custom font and custom font size to overcome it.

**Reply by kirill-d-lappo** [reply]
so, it seems any modifier in the clock breaks the illusion of replacing system clocks. oh, boy, when will MS do anything on this?...
![image](https://user-images.githubusercontent.com/11411852/146656583-71882abc-c901-496e-b5c9-30da983566cd.png)

**Reply by Razorwindmo** [reply]
Same issue here with overlapping the Windows clock
![Untitled](https://user-images.githubusercontent.com/65815231/151781196-690b0d7c-57c6-46d4-9116-3cad7e05bc6d.jpg)

**Reply by marticliment** [reply]
@Razorwindmo, try deleting the .elevenclock folder placed in your user directory. This should already be fixed.

**Comment by Razorwindmo** [comment]
Thank you. That did work but, if it was supposed to be fixed I have bad news. The folder was there and I deleted it and all is well now. Version 3.2.0

Thank you,
Jody Treft
 

 
 

    On Monday, January 31, 2022, 06:39:20 AM CST, Martí Climent ***@***.***> wrote:  
 
 


@Razorwindmo, try deleting the .elevenclock folder placed in your user directory. This should already be fixed.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Razorwindmo** [comment]
I just woke up this morning to find that the issue with elevenclock and its folder has reappeared in my user folder without my intervention.

Thank you,
Jody Treft
 

 
 

    On Monday, January 31, 2022, 06:39:20 AM CST, Martí Climent ***@***.***> wrote:  
 
 


@Razorwindmo, try deleting the .elevenclock folder placed in your user directory. This should already be fixed.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by marticliment** [reply]
Can you please share your log?

**Comment by Razorwindmo** [comment]
Now, this is after removing the folder and it is working.

---------------------------------------------------------------------------------------------------
   ElevenClock's 3.2.0 (v3.2) log: Select all the text and hit Ctrl+C to copy it   All modules loaded successfully and sys.stdout patched correctly, starting main script   Translator function set language to "en"
---------------------------------------------------------------------------------------------------
 Log legend: 🔵: Verbose 🟢: Information 🟡: Warning 🟠: Handled unexpected exception 🔴: Unhandled unexpected exception 🟣: Handled expected exception
🔵 Initializing clock 0...🔵 Using taskbar background color🟢 Regular sized taskbar🔵 Monitor geometry: PyQt5.QtCore.QRect(0, 0, 2560, 1440)🟡 Taskbar at bottom🟢 Clock on the right🟢 Using qt's default positioning system🔵 Clock geometry: PyQt5.QtCore.QRect(2360, 1392, 200, 48)🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']🔵 Font size: 9.3🟢 Using white text (dark mode)🔵 Full screen rect:  (0, 0, 2560, 1440)🟣 Expected AttributeError on backgroundLoop thread🟢 Loaded clocks in 0.9879584312438965🔵 Show/hide loop started with parameters: HideonFS:True, NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:True, DisableNotifications:True, legacyFullScreenMethod:False🔵 Restart loop: 1🟢 Cached text input host hwnd: 66584🟣 Expected AttributeError on backgroundLoop thread🔵 System tray icon enabled🔵 Starting update check🔵 Low cpu mode is set to False. DisableNotifications is set to False🟢 Loaded everything in 1.560426950454712🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver🟢 Updates not found🔵 Starting update check🔵 Version URL: https://versions.somepythonthings.tk/versions/elevenclock.ver🟢 Updates not found🔵 Announcement URL: http://www.somepythonthings.tk/resources/elevenclock.announcement🔵 Image URL: https://www.somepythonthings.tk/resources/we-are-on-twitter.webp🔵 Copying log to the clipboard...



Thank you,
Jody Treft
 

 
 

    On Thursday, February 3, 2022, 08:44:33 AM CST, Martí Climent ***@***.***> wrote:  
 
 


Can you please share your log?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
